lib/spack/docs/workflows.rst

```diff
@@ -1090,6 +1090,211 @@ or filesystem views.  However, it has some drawbacks:
    integrate Spack explicitly in their workflow.  Not all users are
    willing to do this.
 
+-------------------------------------
+Using Spack to Replace Homebrew/Conda
+-------------------------------------
+
+Spack is an incredibly powerful package manager, designed for supercomputers
+where users have diverse installation needs. But Spack can also be used to
+handle simple single-user installations on your laptop. Most macOS users are
+already familiar with package managers like Homebrew and Conda, where all
+installed packages are symlinked to a single central location like ``/usr/local``.
+In this section, we will show you how to emulate the behavior of Homebrew/Conda
+using :ref:`<environments>`!
+
+^^^^^
+Setup
+^^^^^
+
+First, let's create a new environment. We'll assume that Spack is already set up
+correctly, and that you've already sourced the setup script for your shell.
+To create a new environment, simply run:
+
+.. code-block:: console
+
+   $ spack env create myenv
+   ==> Updating view at /Users/me/spack/var/spack/environments/myenv/.spack-env/view
+   ==> Created environment 'myenv' in /Users/me/spack/var/spack/environments/myenv
+   $ spack env activate myenv
+
+Here, *myenv* can be anything you want to name your environment. Next, we can add
+a list of packages we would like to install into our environment. Let's say we
+want a newer version of Bash than the one that comes with macOS, and we want a
+few Python libraries. We can run:
+
+.. code-block:: console
+
+   $ spack add bash
+   ==> Adding bash to environment myenv
+   ==> Updating view at /Users/me/spack/var/spack/environments/myenv/.spack-env/view
+   $ spack add python@3:
+   ==> Adding python@3: to environment myenv
+   ==> Updating view at /Users/me/spack/var/spack/environments/myenv/.spack-env/view
+   $ spack add py-numpy py-scipy py-matplotlib
+   ==> Adding py-numpy to environment myenv
+   ==> Adding py-scipy to environment myenv
+   ==> Adding py-matplotlib to environment myenv
+   ==> Updating view at /Users/me/spack/var/spack/environments/myenv/.spack-env/view
+
+Each package can be listed on a separate line, or combined into a single line.
+Notice that we're explicitly asking for Python 3 here. You can use any spec
+you would normally use on the command line.
+
+Next, we want to manually configure a couple of things. In the ``myenv``
+directory, we can find the ``spack.yaml`` that actually defines our environment.
+
+.. code-block:: console
+
+   $ vim ~/spack/var/spack/environments/myenv/spack.yaml
```

Comment:
I wish I could put this file in my `~/.spack` directory so I could version control it with the rest of my configs...

Reply:
I actually ended up storing the `spack.yaml` in my `~/.spack` directory anyway. As long as I `cd` to this directory, Spack automatically puts me in the correct environment. I don't even need to source the `setup_parser.sh` to run `spack env` commands. But yes, your symlink solution should work fine. For the purposes of the docs, let's just keep it simple.